server/configuration.js

```diff
@@ -39,4 +46,7 @@ module.exports = {
 
     /** Blocked Tools. A comma-separated list of tools that should not appear on boards. */
     BLOCKED_TOOLS: (process.env['WBO_BLOCKED_TOOLS'] || "").split(','),
+
+    /** Automatically switch to White-out on finger touch after drawing with Pencil using a stylus */
+    AUTO_FINGER_WHITEOUT: parseBool(process.env['AUTO_FINGER_WHITEOUT']),
```

Comment:
Enabling it won't have any impact on any device but the iPad, right ? And you say it's the most intuitive on the iPad ? If so, I think we can enable it by default.

```suggestion
    AUTO_FINGER_WHITEOUT: process.env['AUTO_FINGER_WHITEOUT'] !== "disabled",
```

Very few people change the defaults.